paper/paper.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'splot: Visual Analytics for Spatial Statistical Analysis in PySAL'
+title: '`splot` - visual analytics for spatial statistics'
```

Comment:
@slumnitz will the title not fit on one line with "PySAL?" I think we should try (but you are ultimately the final decision maker). If it doesn't fit the we can simply make another push without it.

Reply:
what do you think?